### PR TITLE
[86bzt774x][color-picker] some a11y improvements

### PR DESCRIPTION
**Pull request by ilyabrower**
## Motivation and Context
I've changed some aria-* attributes and roles in ColorPicker components.

## How has this been tested?
Manually

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [x] Nice improve.

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly or it's not required.
- [x] Unit tests are not broken.
- [x] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

File: semcore/color-picker/src/components/Item.tsx [hunk 1]
```diff
@@ -52,9 +66,14 @@ export function Item(props: ItemAsProps) {
       {displayLabel && <SLabel data-value={value || '#6C6E79'}>A</SLabel>}
       <Children />
       {editable && (
-        <SCloseIcon onClick={onRemove}>
-          <CloseM color='gray-500' w={10} h={10} />
-        </SCloseIcon>
+        <>
+          <SCloseIcon tabIndex={-1} aria-hidden={true} onClick={onRemove}>
+            <CloseM color='gray-500' w={10} h={10} />
```
Comment (msereniti) [inline semcore/color-picker/src/components/Item.tsx:71]: If it changed, maybe we should change color to design token here?

**Review by msereniti — APPROVED** [review]
Let's change the color to design token before merging

File: semcore/color-picker/src/components/Item.tsx (hunk 1 above)
Comment (j-mnizhek) [reply semcore/color-picker/src/components/Item.tsx:71]: Yes! To `icon-primary-neutral`, as in Figma

File: semcore/color-picker/src/components/colors.tsx [hunk 2]
```diff
@@ -33,18 +38,26 @@ export function Colors(props: ColorsAsProps) {
...
+        aria-label={getI18nText('customColors')}
+      >
+        {Children.origin ? (
+          <Children />
+        ) : (
+          colors?.map((color) => <PaletteManager.Item value={color} key={color} />)
+        )}
+      </SColors>
+      <SPlusButton onClick={onPlusButtonClick} role='button' aria-label={getI18nText('addColor')}>
         <MathPlusM color='gray-500' />
```
Comment (j-mnizhek) [inline semcore/color-picker/src/components/colors.tsx:59]: Let's update the color to `icon-primary-neutral`, as in Figma 🙏

File: semcore/color-picker/src/components/Item.tsx (hunk 1 above)
Comment (ilyabrower) [reply semcore/color-picker/src/components/Item.tsx]: fixed

File: semcore/color-picker/src/components/colors.tsx (hunk 2 above)
Comment (ilyabrower) [reply semcore/color-picker/src/components/colors.tsx]: fixed

**Review by j-mnizhek — APPROVED** [review]
Pushed small update to:
- the ColorPicker's trigger size
- updated the size of the labels above the inputs in the ColorPicker examples

The rest looks ggod 👍

File: semcore/color-picker/src/translations/en.json [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
-  "emptyColor": "Color field, empty",
-  "goToPaletteHint": "{currentColor}, press Tab to go to palette or click to hide palette",
-  "palette": "Colors palette, press Tab+Shift to go back to color field",
+  "emptyColor": "Color field",
+  "palette": "Colors palette",
   "presetColors": "Preset colors",
-  "customColors": "Custom color field container",
-  "itemColor": "Color {color}",
+  "customColors": "Custom preset colors",
+  "addColor": "Add color",
```
Comment (msereniti) [inline semcore/color-picker/src/translations/en.json:7]: Did you submit it to translators?

**Pull request merged**